frontend/src/metabase/visualizations/components/LeafletChoropleth.jsx

```diff
@@ -75,29 +78,29 @@ const LeafletChoropleth = ({
       // main layer
       L.featureGroup([
         L.geoJson(geoJson, {
-          style: style,
-          onEachFeature: onEachFeature,
+          style,
+          onEachFeature,
         }),
       ]).addTo(map);
 
-      // left and right duplicates so we can pan a bit
-      L.featureGroup([
-        L.geoJson(geoJson, {
-          style: style,
-          onEachFeature: onEachFeature,
-          coordsToLatLng: ([longitude, latitude]) =>
-            L.latLng(latitude, longitude - 360),
-        }),
-        L.geoJson(geoJson, {
-          style: style,
-          onEachFeature: onEachFeature,
-          coordsToLatLng: ([longitude, latitude]) =>
-            L.latLng(latitude, longitude + 360),
-        }),
-      ]).addTo(map);
+      // // left and right duplicates so we can pan a bit
```

Comment:
Remove the map duplicates.

Reply:
Any reason this code is only commented out and not deleted?

Do we rely on the duplicates to do panning somewhere (as the comment states)?

@flamber Do you some ideas on the above?

Reply:
@ariya I don't believe we do (anymore) because all interactive options we set to `false`.
Would be great to have @flamber expertise on that.